paimon-web-ui/src/views/metadata/components/columns-form/index.tsx

```diff
@@ -115,16 +135,20 @@ export default defineComponent({
         <n-card
           bordered={true}
           title={`${this.isEdit ? 'Edit' : 'Create'} Column`}
-          style="width: 1100px"
+          style={{ width: this.isEdit ? '1000px' : '1110px' }}
         >
           {{
-            'header-extra': () => (
-              <n-button quaternary circle size="tiny" onClick={this.handleAddOption}>
-                <n-icon>
-                  <Add />
-                </n-icon>
-              </n-button>
-            ),
+            'header-extra': () => {
+              if (!this.isEdit) {
+                return (
+                  <n-button quaternary circle size="tiny" onClick={this.handleAddOption}>
+                    <n-icon>
+                      <Add />
+                    </n-icon>
+                  </n-button>
+                )
+              }
+            },
```

Comment:
Please support the new column when editing.

Reply:
Currently, the backend interface does not support adding columns when editing, but only supports modifying existing columns. If necessary, you can modify the backend interface later.